### PR TITLE
Reorganize the code a bit

### PR DESCRIPTION
**Pull request by siraben**

### CODE REVIEW

**Comment by NilsIrl** [comment]
Please rebase your code on trunk to make use of the added github action (black code formatter)

File: bot.py [hunk 1]
```diff
@@ -172,10 +151,7 @@ async def on_raw_reaction_add(self, payload: discord.RawReactionActionEvent):
         if payload.user_id == self.user.id:
             return
 
-        # TODO: this could be avoided when the emoji isn't one of those we handle
```
Comment (NilsIrl) [inline bot.py:176]: Why remove this comment?

File: bot.py [hunk 2]
```diff
@@ -42,87 +40,68 @@ async def attachment_to_langs(attachment, model_name=None):
...
-            await interaction.followup.edit_message(
-                interaction.message.id,
-                content=await attachment_to_text(
-                    audio_message.attachments[0],
-                    model_name=self.model_size,
-                    language=self.language_selected,
-                ),
-                view=self,
+    async def on_language_selected(self, interaction: discord.Interaction):
+        self.language_selected = self.children[0].values[0]
```
Comment (NilsIrl) [inline bot.py:80]: Not sure this is better (~~tbh would say this about most of this PR~~)

**Review by NilsIrl — COMMENTED** [review]
After the code formatter has run a lot of the changes will be reverted.

For the changes to ModelSelectorView. Not sure it's better than how it was before. Now if components are added, the hard-coded children indices will need to be changed.

tbh I doubt there's any changes that will be strictly better (although I'm happy with non-strictly better changes too).

**Comment by siraben** [comment]
This is just to separate out the callbacks from ModelSelectorView to make the code more modular for future changes, not strictly necessary.

**Comment by NilsIrl** [comment]
Makes code less readable.

**Pull request closed without merging**